widgets/src/main/java/org/pentaho/gwt/widgets/client/genericfile/GenericFileNameUtils.java

```diff
@@ -34,15 +34,11 @@ public class GenericFileNameUtils {
   private GenericFileNameUtils() {
   }
 
-  // TODO: For repo, NameUtils.encodeRepositoryPath, ~ is converted to \t...
-  // Here, theres pvfs:// prefixes in play as well...
-  // See Common-UI's Encoder.encodeRepositoryPath.
   @NonNull
-  public static String encodePath( @NonNull String path ) {
-    return path
-      .replace( ":", "~" )
-      .replace( PATH_SEPARATOR, ":" );
-  }
+  public static native String encodePath( @NonNull String path )
```

Comment:
I don't think there is anyway to test GWT JSNI code. So I won't enforce unit testing